src/io/sampling.jl

```diff
@@ -142,25 +223,17 @@ function sample_guided(d::Domain, n::Int, sampler=SobolSample())::DataFrame
 
     # Remove unguided scenarios as an option
     mod_df = copy(spec_df)
-    guided_col = mod_df.fieldname .== :guided
-    g_upper = mod_df[guided_col, :upper_bound][1]
 
     insertcols!(mod_df, :val, :bounds => copy([d.model[:bounds]...]))
-    mod_df[guided_col, :val] .= 1
-    mod_df[guided_col, :lower_bound] .= 1
-    mod_df[guided_col, :bounds] .= [(1, g_upper)]
-    mod_df[guided_col, :full_bounds] .= [(1, g_upper)]
+    mod_df = _adjust_guided_lower_bound(mod_df, 1)
 
     # Sample without unguided, then revert back to original model spec
     ADRIA.update!(d.model, mod_df)
-    samples = _sample(d, n, sampler)
-    samples = adjust_samples(d, samples)
+    samples = adjust_samples(d, sample(d, n, sampler))
 
     # Note: updating with spec_df does not work.
-    mod_df[guided_col, :val] .= 0
-    mod_df[guided_col, :lower_bound] .= 0
-    mod_df[guided_col, :bounds] .= [(0, g_upper)]
-    mod_df[guided_col, :full_bounds] .= [(0, g_upper)]
+    mod_df = _adjust_guided_lower_bound(mod_df, 0)
```

Comment:
Found a bug: this should be -1 (reverting back to the original value) but let me fix it on my end.